systemtest/src/test/java/io/strimzi/systemtest/KafkaClusterTest.java

```diff
@@ -306,4 +308,22 @@ public void testForUpdateValuesInConfigMap() {
             assertEquals("23", getValueFromJson(zkPodJson, initialDelaySecondsPath));
         }
     }
+
+    @Test
+    @KafkaCluster(name = "my-cluster", kafkaNodes = 3, config = {
+            @CmData(key = "KAFKA_DEFAULT_REPLICATION_FACTOR", value = "1")})
+    @Topic(name = "test-topic", clusterName = "my-cluster")
+    public void testSendMessages() {
+        String topicName = "test-topic";
+        String clusterName = "my-cluster";
+        int messagesCount = 5;
+        List<String> messagesToSend = new ArrayList<>();
+        for (int i = 0; i < messagesCount; i++) {
+            messagesToSend.add("Test message " + i);
+        }
+        sendMessages(messagesToSend, clusterName, topicName);
```

Comment:
As for me, the first argument should be `String...` (Arbitrary Number of Arguments) because we can have a test case when we will send only one message and creating new List for that may be excessive action.

Reply:
I've created separate methods for sending String and List of messages.
WDYT @serrss